Libplanet.Tools/Apv.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.IO;
+using Bencodex;
+using Bencodex.Types;
+using Cocona;
+using Libplanet.Crypto;
+using Libplanet.Net;
+
+namespace Libplanet.Tools
+{
+    public class Apv
+    {
+        [Command(Description = "Sign a new app protocol version.")]
+        public void Sign(
+            [Argument(Name = "KEY-ID", Description = "A private key to use for signing.")]
+            Guid keyId,
+            [Argument(Name = "VERSION", Description = "A version number to sign.")]
+            int version,
+            [Option(
+                'p',
+                ValueName = "PASSPHRASE",
+                Description = "Take passphrase through this option instead of prompt."
+            )]
+            string? passphrase = null,
+            [Option(
+                'E',
+                ValueName = "FILE",
+                Description = "Bencodex file to use for extra data.  " +
+                    "For standard input, use a hyphen (`-').  " +
+                    "For an actual file named a hyphen, prepend `./', i.e., `./-'."
+            )]
+            string? extraFile = null,
+            [Option(
+                'e',
+                ValueName = "KEY=VALUE",
+                Description = "Set a value to a key on extra Bencodex dictionary.  " +
+                    "Can be applied multiple times (e.g., `-e foo=1 -e bar=baz').  " +
+                    "This option implies the extra data to be a Bencodex dictionary, " +
+                    "hence cannot be used together with -E/--extra-file option."
+            )]
+            string[]? extra = null
+        )
+        {
+            PrivateKey key = new Key().UnprotectKey(keyId, passphrase);
+            IValue? extraValue = null;
+            if (extraFile is string path)
+            {
+                if (extra is string[] e && e.Length > 0)
+                {
+                    throw Utils.Error(
+                        "-E/--extra-file and -e/--extra cannot be used together at a time."
+                    );
+                }
+
+                var codec = new Codec();
+                if (path == "-")
+                {
+                    // Stream for stdin does not support .Seek()
+                    using MemoryStream buffer = new MemoryStream();
+                    using (Stream stream = Console.OpenStandardInput())
```

Comment:
Can we write as below? or there is any other reason?
```suggestion
                    using Stream stream = Console.OpenStandardInput();
```

Reply:
As `stream` is no more used after `stream.CopyTo(buffer)`, an explicit scope was used by intention.